lib/standards/html-elms.js

```diff
@@ -674,7 +674,7 @@ const htmlElms = {
     noAriaAttrs: true
   },
   picture: {
-    contentTypes: ['embedded', 'phrasing', 'flow'],
+    contentTypes: ['phrasing', 'flow'],
```

Comment:
```suggestion
    // In HTML, this is considered "embedded". For axe-core we don't want this since its children will not be hidden
    contentTypes: ['phrasing', 'flow'],
```